tox.ini

```diff
@@ -0,0 +1,63 @@
+[tox]
+envlist =
+    {
+    py 37,py 38,py36}-tests,check-lint,docs
```

Comment:
strange formatting (and spaces) :)

Reply:
Hm, might have to fix a thing or two in the cookiecutter...

Reply:
A logical consequence is that .travis.yml is equally broken.